contracts/base/BaseLiquidityManagement.sol

```diff
@@ -168,17 +168,8 @@ abstract contract BaseLiquidityManagement is SafeCallback, IBaseLiquidityManagem
         bool claims,
         BalanceDelta tokensOwed
     ) external returns (BalanceDelta delta) {
-        BalanceDelta feeDelta = poolManager.modifyLiquidity(
-            key,
-            IPoolManager.ModifyLiquidityParams({
-                tickLower: params.tickLower,
-                tickUpper: params.tickUpper,
-                liquidityDelta: 0
-            }),
-            hookData
-        );
-
-        poolManager.modifyLiquidity(key, params, hookData);
+        BalanceDelta feeDelta;
+        (delta, feeDelta) = poolManager.modifyLiquidity(key, params, hookData);
```

Comment:
by returning the fee delta, the position manager does not need to first claim fees and then increase liquidity in separate calls

saves about 12k gas